src/main/java/seedu/duke/Storage.java

```diff
@@ -0,0 +1,86 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Storage {
+
+    private static final File save = new File("./data/duke.txt");
+
+    public static boolean saveExists() {
+        return save.exists();
+    }
```

Comment:
how about constructing a storage instance using a file path specified by the main application instead of fixing the file path?